src/Core/Util/String/StringModifierInterface.php

```diff
@@ -28,6 +28,9 @@
 
 /**
  * Defines reusable methods for strings modifications.
+ *
+ * @method str2url(string $string, bool $allowAccentedChars) will be added in 9.0
```

Comment:
I didn't know this notation, good to know!!

Reply:
me neither 😊